src/base/bittorrent/session.h

```diff
@@ -29,51 +29,21 @@
 
 #pragma once
 
-#include <variant>
-#include <vector>
-
-#include <libtorrent/add_torrent_params.hpp>
-#include <libtorrent/fwd.hpp>
-#include <libtorrent/torrent_handle.hpp>
-
-#include <QHash>
-#include <QPointer>
-#include <QSet>
 #include <QtContainerFwd>
-#include <QVector>
+#include <QObject>
 
-#include "base/path.h"
-#include "base/settingvalue.h"
-#include "base/types.h"
+#include "base/pathfwd.h"
 #include "addtorrentparams.h"
 #include "cachestatus.h"
 #include "categoryoptions.h"
 #include "sessionstatus.h"
 #include "torrentinfo.h"
 #include "trackerentry.h"
```

Comment:
I think some of those headers can be forward declared.

Reply:
Or maybe they are all required...

Reply:
This will most likely affect many sources that include this header. I didn't intend to do it since it isn't required for the job. It can be done later.